aws-auth/src/main/java/software/amazon/awssdk/awsauth/AwsExecutionAttributes.java

```diff
@@ -32,39 +34,14 @@
                      + "4. The remaining public AWS stuff should be moved to the AWS module. "
                      + "We should also consider making some of the SDK/AWS-owned set of attributes part of the immutable context "
                      + "if we don't want the interceptors to modify them.")
-public final class AwsExecutionAttributes {
+public interface AwsExecutionAttributes extends SdkExecutionAttributes {
```

Comment:
This seems to be more of an awscore thing rather than auth specific.

Reply:
This is used by Aws4Signer for now to get the credentials from ExecutionAttributes. I guess this will be moved around once we start working on signer refactor.

Reply:
Shirley the awsauth module should depend on awscore?